docs/source/60_building_apps.md

```diff
@@ -284,3 +284,327 @@ class MyCommandKryoRegistrator extends KryoRegistrator {
   }
 }
 ```
+
+## Using ADAM as a library {#library}
+
+The `adam-core` and `adam-apis` modules are designed to be used as a library in
+downstream applications. Scala apps should use `adam-core`, while Java
+applications should also depend on `adam-apis`. The main entrypoint to ADAM is
+the [ADAMContext](#adam-context), which allows genomic data to be loaded in to
+Spark as [GenomicRDD](#genomic-rdd). GenomicRDDs can be transformed using ADAM's
+built in [pre-processing algorithms](#algorithms), [Spark's RDD
+primitives](#transforming), and ADAM's [pipe](#pipes) APIs.
+
+### Accessing pre-built libraries
+
+These libraries are available from [Maven Central](http://search.maven.org) with
+the following coordinates:
+
+```
+<dependency>
+    <groupId>org.bdgenomics.adam</groupId>
+    <artifactId>adam-core${binary.version}</artifactId>
+    <version>0.21.0</version>
```

Comment:
would prefer a `${version}` or `${adam.version}` variable here, so that the docs don't need to be updated on every new release